pkg/rpmdb_test.go

```diff
@@ -95,23 +84,8 @@ func TestPackageList(t *testing.T) {
 			got, err := db.ListPackages()
 			require.NoError(t, err)
 
-			// They are tested in another function.
```

Comment:
What if re-creating `PackageInfo`?

```
for i, g := range got {
	got[i] = PackageInfo {
		Epoch: g.Epoch,
		Name: g.Name
		Version: g.Version,
		Release: g.Release,
		Arch: g.Arch,
		...
	}
}

Reply:
I don't understand what you're suggesting here.

Reply:
> I don't understand what you're suggesting here.

Never mind. I think I understand what you're getting at. Let me give this another shot.